configure

```diff
@@ -553,16 +558,17 @@ detect_resolv_static()
 	TMPFILE=$(mktemp /tmp/bitlbee-configure.XXXXXX)
 	ret=1
 	for i in $systemlibdirs; do
-		if [ -f $i/libresolv.a ]; then
-			echo "$RESOLV_TESTCODE" | $CC -o $TMPFILE -x c - -Wl,$i/libresolv.a >/dev/null 2>/dev/null
+		if [ -f "$i/libresolv.a" ]; then
+			echo "$RESOLV_TESTCODE" | $CC -o "$TMPFILE" -x c - -Wl,"$i/libresolv.a" >/dev/null 2>/dev/null
+			# shellcheck disable=SC2181
```

Comment:
can we just globally disable these, rather than having to sprinkle them around the code wherever they appear?

Reply:
Yes, changed.